.github/workflows/build.yml

```diff
@@ -54,7 +54,7 @@ jobs:
           submodules: recursive
 
       - name: Linux Get dependencies
-        run: sudo apt install -y cpanminus build-essential libncurses5-dev libreadline-dev libssl-dev perl
+        run: sudo apt install -y cpanminus build-essential libncurses5-dev libreadline-dev libssl-dev perl libssl-dev
```

Comment:
why we need this? only for testing? or the apisix need it?

Reply:
It seems duplicate? The original install list already got `libssl-dev`

Reply:
Removed.